### PR TITLE
Add instructions on how to run the demo

### PR DESCRIPTION
**Pull request by MusaTalluzi**

### CODE REVIEW

**Comment by rsynek** [comment]
@Fonifan would you please review the instructions?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+- Download kie-server-...-ee7.war and place it into `WILDFLY_HOME/standalone/deployments` as `kie-server.war`.
+- Configure user(s) and role(s). Execute the following command `WILDFLY_HOME/bin/add-user.sh -a -u 'kieserver' -p 'kieserver1!' -ro 'kie-server'`. You can of course choose different username and password, just make sure that the user has role `kie-server`
+- Start the server by running `WILDFLY_HOME/bin/standalone.sh -c standalone-full.xml`. The default location for kieserver is `http:/localhost:8080/kie-server/services/rest/server`. Check this location in a web browser to verify the server is running. You will be prompted to type in username and password, use the ones you chose when adding a new user.
+
+### Now that the kieserver is running, let's deploy an OptaPlanner service to solve a task assignment problem.
+- Copy the OptaPlanner service kjar to your local mvn repository, this can be done by typing the command `mvn install` in `kie-server-optaplanner-task-assignment/task-assignment-kjar`.
+- Run the UI:
+  - Go to `kie-server-optaplanner-task-assignment/task-assignment-kjar`
```
Comment (yurloc) [inline README.md]: `s/kjar/ui/`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,24 @@
...
+  - `npm install` to install the dependencies.
+  - `npm start` this will start the frontend application. If it doesn't open in a default browser, open your favorite browswer and go to `localhost:3000`
```
Comment (yurloc) [inline README.md]: Typo: browswer.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,24 @@
...
+
+# Running the demo
+### First start a Wildfly instance and deploy KIE artifacts:
+- Download and unzip the WildFly distribution. Let’s call the root of the distribution WILDFLY_HOME. This directory is named after the WildFly version, so for example `wildfly-11.0.1.Final` is the one we're using for this demo.
+- Download kie-server-...-ee7.war and place it into `WILDFLY_HOME/standalone/deployments` as `kie-server.war`.
+- Configure user(s) and role(s). Execute the following command `WILDFLY_HOME/bin/add-user.sh -a -u 'kieserver' -p 'kieserver1!' -ro 'kie-server'`. You can of course choose different username and password, just make sure that the user has role `kie-server`
+- Start the server by running `WILDFLY_HOME/bin/standalone.sh -c standalone-full.xml`. The default location for kieserver is `http:/localhost:8080/kie-server/services/rest/server`. Check this location in a web browser to verify the server is running. You will be prompted to type in username and password, use the ones you chose when adding a new user.
+
+### Now that the kieserver is running, let's deploy an OptaPlanner service to solve a task assignment problem.
```
Comment (yurloc) [inline README.md]: Consistency: KIE Server.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,24 @@
 # optatask
 OptaPlanner service for task assignment
+
+# Requirements
+- JDK 1.8 or above
+- Maven 3.0+
+- node.js 8.12+
+- npm 6.4.1+
+- [Wildfly](https://download.jboss.org/wildfly/11.0.0.Final/wildfly-11.0.0.Final.zip)
```
Comment (yurloc) [inline README.md]: Let's use "WildFly" consistently.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,24 @@
...
+- KIE Server war file from [OptaPlanner website](http://www.optaplanner.org/download/download.html) (Execution server -> Any application server, unzip to find the war file)
+
+# Running the demo
+### First start a Wildfly instance and deploy KIE artifacts:
```
Comment (yurloc) [inline README.md]: Skipping a section level.

File: README.md [hunk 6]
```diff
@@ -1,8 +1,29 @@
...
+- JDK 1.8 or above
+- Maven 3.0+
+- node.js 8.12+
+- npm 6.4.1+
+- [WildFly](https://download.jboss.org/wildfly/11.0.0.Final/wildfly-11.0.0.Final.zip)
+- KIE Server war file from [OptaPlanner website](http://www.optaplanner.org/download/download.html) (Execution server -> Any application server, unzip to find the war file)
+
+## Running the demo
+### First start a WildFly instance and deploy KIE artifacts:
+- Download and unzip the WildFly distribution. Let’s call the root of the distribution WILDFLY_HOME. This directory is named after the WildFly version, so for example `wildfly-11.0.1.Final` is the one we're using for this demo.
```
Comment (yurloc) [inline README.md]: WildFly 11 is pretty old. Let's use the latest (15.0.1 works for me).

File: README.md [hunk 7]
```diff
@@ -1,8 +1,29 @@
...
+- Download kie-server-...-ee7.war and place it into `WILDFLY_HOME/standalone/deployments` as `kie-server.war`.
+- Configure user(s) and role(s). Execute the following command `WILDFLY_HOME/bin/add-user.sh -a -u 'kieserver' -p 'kieserver1!' -ro 'kie-server'`. You can of course choose different username and password, just make sure that the user has role `kie-server`
+- Start the server by running `WILDFLY_HOME/bin/standalone.sh -c standalone-full.xml`. The default URI for KIE Server is `http:/localhost:8080/kie-server/services/rest/server`. Check this location in a web browser to verify the server is running. You will be prompted to type in username and password, use the ones you chose when adding a new user.
```
Comment (yurloc) [inline README.md]: Maybe remove the backticks around KIE server URL? It's more convenient to have a link that I can click than having to copy & paste it.

File: README.md [hunk 8]
```diff
@@ -1,8 +1,48 @@
...
+cd task-assignment-kjar
+mvn install
+```
+2. Run the kie server:
+```bash
+cp kie-server-...-ee7.war WILDFLY_HOME/standalone/deployments/kie-server.war
+WILDFLY_HOME/bin/add-user.sh -a -u 'kieserver' -p 'kieserver1!' -ro 'kie-server'
+WILDFLY_HOME/bin/standalone.sh -c standalone-full.xml
+```
+1. Run the UI:
```
[the remark this answers is not in the document]
Comment (MusaTalluzi) [reply README.md]: Yes it is! Thanks for the catch

**Pull request merged**